tcms/templates/datetimepicker_script.html

```diff
@@ -1,13 +0,0 @@
-<script>
```

Comment:
File is deleted but the new file in `tcms/templates/include/datetimepicker_script.html` is not added to this pull request.